src/lib/fetch.ts

```diff
@@ -218,34 +245,6 @@ function stringifyAccount(account: FlexibleAccount): FetchedAccount {
   };
 }
 
-async function checkResponseStatus(
```

Comment:
this code block was just moved to the bottom of the file. I often move unimportant / helper-type code to the bottom, and  the exported API / most important parts to the top, because I think this makes code more readable